Firmware/Marlin_main.cpp

```diff
@@ -11024,7 +11069,7 @@ void restore_print_from_ram_and_continue(float e_move)
 	set_destination_to_current();
 
     restore_print_file_state();
-
+    eeprom_update_byte((uint8_t*)EEPROM_UVLO, PowerPanic::NO_PENDING_RECOVERY);
 	lcd_setstatuspgm(MSG_WELCOME);
     saved_printing_type = PowerPanic::PRINT_TYPE_NONE;
 	saved_printing = false;
```

Comment:
Optimisation idea: Can we use `cancel_saved_printing()` here?

So change:

```c
    eeprom_update_byte((uint8_t*)EEPROM_UVLO, PowerPanic::NO_PENDING_RECOVERY);
	lcd_setstatuspgm(MSG_WELCOME);
    saved_printing_type = PowerPanic::PRINT_TYPE_NONE;
	saved_printing = false;
```

to this:

```c
    cancel_saved_printing();
	lcd_setstatuspgm(MSG_WELCOME);
```

Reply:
No as `cancle_saved_printing()` sets `saved_start_position[0] = SAVED_START_POSITION_UNSET;` and the function 
`void restore_print_from_ram_and_continue(float e_move)` only resets some states BUT continues to print.